### ISSUE TITLE
Annotations may get duplicated when using assembler

### ISSUE THREAD

**Issue by Geolykt** [Col-E/Recaf#370]
**Describe the bug**

If I save the assembler window multiple times, then the annoation of the method gets duplicated.
I guess duplicate annotations could get stripped, since they appear to be illegal bytecode see blow; however I am not an expert in this topic

```
java.lang.annotation.AnnotationFormatError: Duplicate annotation for class: interface org.bukkit.event.EventHandler: @org.bukkit.event.EventHandler(priority=NORMAL, ignoreCancelled=false)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations2(AnnotationParser.java:126) ~[?:?]
	at sun.reflect.annotation.AnnotationParser.parseAnnotations(AnnotationParser.java:73) ~[?:?]
	at java.lang.reflect.Executable.declaredAnnotations(Executable.java:604) ~[?:?]
	at java.lang.reflect.Executable.declaredAnnotations(Executable.java:602) ~[?:?]
	at java.lang.reflect.Executable.getAnnotation(Executable.java:572) ~[?:?]
	at java.lang.reflect.Method.getAnnotation(Method.java:695) ~[?:?]
	<snip>
```

**Comment by Col-E** [comment]
Can you supply a class file that demonstrates this behavior? Swore there was a recent issue about this that got resolved 🤔

**Comment by Geolykt** [comment]
me/SpookyproYT/welcomer/events/ConnectionEvents.class within https://www.spigotmc.org/resources/welcomer.90362/download?version=389525

**Comment by Col-E** [comment]
😬 Yeah I'll get right on that

![image](https://user-images.githubusercontent.com/21371686/114280635-3dab8980-9a08-11eb-9ee3-982c2eae02c2.png)

**Comment by Col-E** [comment]
Seems to be fixed with 3c69b23e079397dbfe91ba14c7224bef61a84fc2, release should be out when CI finishes

**Comment by Geolykt** [comment]
K, I'll test it right away
